src/assets/scss/base.scss

```diff
@@ -367,7 +367,7 @@ button.reserve-submit-btn {
   display: none !important;
 }
 
-/* TODO: Move this somewhere better! */
+/* FUTURE: Move this somewhere better! */
```

Comment:
Would `layout.scss` make sense? 
I dunno!

Not major but another class by the same name exists in `fee-summary.vue` which is a little different than this one.

Reply:
Yes, layout.scss would be an appropriate place for this specific class, but this is tech debt and I don't want to touch it atm :)

Reply:
No prob!